### PR TITLE
Updated steps: expanding content fairly substantially and adding 6th step for post-application support

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card
https://trello.com/c/wo0t9f1S/744-parked-until-step-6-done-upgrade-content-on-steps-to-become-a-teacher
### Context
We found that some of the stripped out content on Steps wasn't sufficient for users' informational needs. This update helps rectify that.
### Changes proposed in this pull request
Can you read through and see if any typos stick out? Links all appear to work in review app. Does Step 2 work for you from a perspective of how users might interpret it? E.g. Does stories *really* need to be there or is it filler?

I've also added a quick update on Ways to Train (sorry not best practice), including making a new heading for other ways to train, which allows me to then add a CTA for our assessment only providers list.

### Guidance to review
Review:

- Steps
- Ways to Train

### CODE REVIEW

File: content/ways-to-train/_assessment_only.html.erb [hunk 1]
```diff
@@ -10,4 +10,8 @@
 
 <p>A number of universities and School Centred Initial Teacher Training programmes (SCITTs) around the country offer Assessment Only routes. Their criteria for entry may vary, and you will need to apply to them directly.</p>
 
-<p>For more information about Assessment Only options, you can view a list of <a href="/assessment-only-providers">approved providers in our directory</a>.
+<p>For more information about Assessment Only options, you can view a list of approved providers in our directory.
```
Comment (peteryates) [inline content/ways-to-train/_assessment_only.html.erb]: While it's not _strictly_ necessary there should be a closing `</p>` tag here

**Review by peteryates — COMMENTED** [review]
Looks good overall, left one tiny comment

File: content/ways-to-train/_assessment_only.html.erb (hunk 1 above)
Comment (MylesJarvis) [reply content/ways-to-train/_assessment_only.html.erb]: Done

**Review by peteryates — APPROVED** [review]
Nice!

**Pull request merged**